packages/date-parser/src/result.js

```diff
@@ -0,0 +1,63 @@
+import { merge } from 'lodash';
+
+const convertToTargetTimezone = (parsedComponent) => {
+  if (!parsedComponent) return null;
+  const offset = parsedComponent.get('timezoneOffset');
+
+  return parsedComponent.dayjs().utcOffset(offset);
```

Comment:
just a quick idea, haven't really thought through, but can we do the followings?
* removing the timezone implier
* setting the timezone directly on the dayjs instance instead of setting the offset
 
My main thought is that if the processing yields correct results for certain scenarios but not for some others, then I think the generic logic should be improved. In this case, the timezone compensation on the results should be good enough that any result is compensated correctly according to the timezone option, regardless of the parsing logic :thinking:

Reply:
Ah i have read the refiner step again and you are already setting the offset according to the result